WPMedia/arc-themes-components
# Themes 1604

Merge https://github.com/WPMedia/arc-themes-components/pull/229 first!!

## Ticket

- [THEMES-1640](https://arcpublishing.atlassian.net/browse/THEMES-1604)

## Description


## Acceptance Criteria


## Test Steps

1. Checkout branch - `git checkout themes-1604`
2. Update dependencies - `npm i`
3. Run fusion local linking your components
4. ... see log for test data

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
